### DISCUSSION TITLE
Will Version 6.1.1 Work In Godot 3.5.1?

### DISCUSSION THREAD

**Discussion by ThatCasual** [Shin-NiL/Godot-Android-Admob-Plugin#209] [Q&A]
Should the most recent release (6.1.1 at the time of writing) be compatible with Godot version 3.5.1? My app crashes whenever the export from Godot 3.5.1 includes this plugin, and I am curious if it may simply not be compatible yet. If not, any news on an update?

**Comment by dpensky** [comment]
> Update to Godot 3.4.5;
> Update to Play Services Ads 21.1.0;
> Update to Android SDK 31.
